internal/auth/oidc/repository_auth_method_update_test.go

```diff
@@ -129,6 +129,75 @@ func Test_UpdateAuthMethod(t *testing.T) {
 				return am
 			},
 		},
+		{
+			name: "null-name-description",
```

Comment:
Please add a test that just unsets a single valueObject like SigningAlg.  I think there is a bug where Update will fail since both the mask and null masks are empty in such a case.

Reply:
ty!